src/main/java/im/status/keycard/KeycardApplet.java

```diff
@@ -36,6 +38,7 @@
   static final byte KEY_PATH_MAX_DEPTH = 10;
   static final byte PAIRING_MAX_CLIENT_COUNT = 1;
   static final byte UID_LENGTH = 16;
+  static final byte SAVED_DATA_SIZE = 255 - SecureChannel.SC_OUT_OFFSET; // Max APDU size less the SC overhead
```

Comment:
This only accounts for the secure channel header, but you also need to account for AES encrypted block size. According to Status, the max payload size of a secure channel APDU is 223 bytes: https://status.im/keycard_api/apdu/opensecurechannel.html

Reply:
998c77d